### PR TITLE
QR Code addition

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** adding QR code content 
- **Is this PR related to a Monolith release?** 2021.08.2

### Resolves:

* [OKTA-417393](https://oktainc.atlassian.net/browse/OKTA-417393)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ curl --request POST \
   --data-urlencode 'device_code=${deviceCode)'
```
Comment (nakuljoshi-okta) [inline packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md:109]: `${deviceCode}`

File: packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md [hunk 2]
```diff
@@ -63,35 +63,35 @@ curl --request POST \
...
 ```json
 {
-   "device_code": "f5ec0430-c2fe-4a93-912d-08631f1effed",
-   "user_code": "HMBSRJRC",
-   "verification_uri": "https://${yourOktaDomain}/activate",
-   "expires_in": 600,
-   "interval": 5
+    "device_code": "4ebdb4de-1f8b-4497-be01-ddfaf83c4e9c",
+    "user_code": "MHXTFRPK",
+    "verification_uri": "https://atko-demo.oktapreview.com/activate",
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md:74]: Why change this? We should stick to using either an example domain or the `${yourOktaDomain}` throughout the guide, shouldn't we?

**Review by larsjohansen-okta — CHANGES_REQUESTED** [review]
Blocking on feedback incorporation

File: packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md:74]: sorry - missed that one - Good catch!

**Review by nakuljoshi-okta — APPROVED** [review]
LGTM

**Pull request merged**